shared/components/themes-head/index.jsx

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
Mmm, indeed. OT: it's going to feel more weird having `themes` in `my-sites` as we put more effort into the logged out experience.